src/database/votePost.js

```diff
@@ -0,0 +1,9 @@
+import { db } from "database";
+import firebase from "firebase";
+
+export const voteForOption = ({ optionName, postId }) => {
+  const increment = firebase.firestore.FieldValue.increment(1);
+  db.collection("posts")
+    .doc(postId)
+    .update({ [optionName]: increment });
```

Comment:
[optionName] is how you use dynamic keys in javascript. Google it to get more clarity if you like